### ISSUE TITLE
Source and confirm most recent ChEMBL-DrugBank mapping file

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#802]
To ensure that as many DrugBank IDs as possible are available on the drug summary page, we need to update the ChEMBL-DrugBank mapping file that the platform-input-support script uses.

- [x] Confirm with ChEMBL that most recent UniChem ChEMBL-DrugBank mapping file is available on the UniChem FTP site
- [x] Download most recent ChEMBL-DrugBank mapping file and upload to otar001-core Google bucket
- [x] Confirm with @cmalangone that most recent file is in correct bucket and labelled correctly for platform-input-support script

**Comment by deniseOme** [comment]
Thanks @andrewhercules for turning my observation/finding into a ticket.

<img width="366" alt="Screen Shot 2020-01-14 at 17 15 57" src="https://user-images.githubusercontent.com/6472381/72366128-93a05d00-36f1-11ea-97ef-5e01b130e157.png">

**Comment by d0choa** [comment]
@andrewhercules could you double check we can get this file? Now that we have a working drug index in Chembl 26

**Comment by andrewhercules** [comment]
@d0choa, CHEMBL have confirmed a new file has been uploaded to [their FTP site](ftp://ftp.ebi.ac.uk/pub/databases/chembl/UniChem/data/wholeSourceMapping/src_id1/) to coincide with the release of CHEMBL26. 

I have downloaded the file and compared to the previous file dated 22 October 2019 and there are 767 new entries.

However, the entries that @deniseOme noted were missing - [Atezolizumab (CHEMBL3707227)](https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL3707227/) and [Durvalumab (CHEMBL3301587)](https://www.ebi.ac.uk/chembl/compound_report_card/CHEMBL3301587/) - do not appear in the file. When checking the CHEMBL site, they also do not list the DrugBank IDs in the Cross References section of the compound profile page.

I will raise with CHEMBL during the next Platform-CHEMBL team meeting. 

However, for 20.02, this issue can be closed as there is an updated file now available.

@cmalangone, I have uploaded the new file - `drugbank-28-01-2020.txt.gz` - into the `otar001-core/DrugBank/annotation` bucket.
